commit 341ad55f2ef50b279d99a8ac7b2ded2d08104d39
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2019-07-16T18:51:11+00:00 (3 months ago)
    
    joyent/bugview#35 JSON endpoint broken by joyent/bugview#33
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/jirapub.js b/jirapub.js
index 8ac0f3a..740e206 100644
--- a/jirapub.js
+++ b/jirapub.js
@@ -408,7 +408,7 @@ handle_issue_json(req, res, next)
 			return;
 		}
 
-		mod_assert.arrayOfStrings(issue.fields.labels, 'labels');
+		mod_assert.arrayOfString(issue.fields.labels, 'labels');
 
 		if (!UNRESTRICTED &&
 		    issue.fields.labels.indexOf(CONFIG.label) === -1) {
